apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -481,4 +481,24 @@ defmodule SiteWeb.ScheduleView do
   def json_safe_route(route) do
     Route.to_json_safe(route)
   end
+
+  @spec retirement_message(Conn.t()) :: Safe.t()
+  def retirement_message(%Conn{assigns: %{route: %Route{type: 2}}}), do: nil
+
+  def retirement_message(conn) do
+    SiteWeb.ViewHelpers.banner_message(conn, :retirement_message)
+  end
+
+  def build_retirement_message(conn) do
```

Comment:
I think *this* is where we'll want to return nil if the route is a commuter rail. Then for that case, when the plug is called it'll see the message returned here is nil and then it won't assign the retirement_message variable.

Reply:
Oh, and I think this is also where we'd want to verify if we're on the relevant tab via `%Plug.Conn{assigns: %{tab: "trip-view"}}`

Reply:
Turns out I realized that we don't even need to create the case for the tab `"trip-view"` because `_trip_view.html.eex` is the view file corresponding to that tab, so no need to pattern-match or check for this.